packages/devextreme/testing/tests/DevExpress.ui.widgets/toolbar.tests.js

```diff
@@ -930,6 +931,76 @@ QUnit.module('adaptivity', moduleConfig, () => {
         assert.strictEqual($visibleItems.length, 3, 'two items was hidden');
     });
 
+    QUnit.test('Buttons declared via template should be hidden if there is no enough free space for them (T1191856)', function(assert) {
+        this.instance.option({
+            items: [
+                {
```

Comment:
[mimor] i'd use const to get rid of duplication

const options = getOptions(location, widget, locateInMenu) =>  {
                    location,
                    widget,
                    locateInMenu,
                    template() {
                        return $('<div />').dxButton({
                            text: 'Button long text'
                        });
                    }
                }

Reply:
https://github.com/DevExpress/DevExtreme/pull/25740/commits/92018c3279c27a839aaf199c2a638b4b487b8861